packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/editable-tile-item/EditableTileItem.spec.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { describe, it } from 'vitest';
+import { ODS_ICON_NAME, ODS_ICON_SIZE } from '@ovhcloud/ods-components';
+import EditableTileItem from './EditableTileItem.component';
+
+describe('EditableTileItem component unit test suite', () => {
+  it('should display a label and the correct icon', () => {
+    // given
```

Comment:
For unit tests it's "Arrange/Act/Assert"
https://www.diffblue.com/resources/how-to-write-better-unit-test-assertions/

Given/When/Then make sense when you have Gherkins